calc/src/App.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: #e5e5e5;
+}
+button {
+  border-radius: 0 0 0 0;
+  width: 72px;
+  height: 62px;
+  background-color: #384351;
+  color: #d1d6db;
+  border: transparent;
+  font-size: 19px;
+}
+
+#App {
+  position: absolute;
```

Comment:
Please try to position it instead of using position, left, top and transformation with flex.